bonnie-businessrules/src/main/java/com/cgi/bonnie/businessrules/order/OrderService.java

```diff
@@ -101,7 +101,9 @@ public boolean setTrackingNumber(long id, String trackingNr) {
     }
 
     public long createOrder(String productId, int quantity, long assignedTo, Status status) {
-        return orderStorage.create(productId, quantity, assignedTo, status);
```

Comment:
@KenyeresKornel can this method deleted?

Reply:
No! Maybe rewrite it. Should be used by the Kafka service to store the incoming orders

Reply:
Sorry, I just noticed the other method you made for that reason. In that case I think it can be deleted.